UnityProject/Assets/Scripts/Core/Chat/DiscordWebhookMessage.cs

```diff
@@ -181,7 +181,7 @@ private void FormatAndSendMessage(string url, Queue<string> queue)
 					break;
 				}
 
-				msg += queue.Dequeue() + "\n";
```

Comment:
Not all messages are in code blocks, so this will need to be kept

Reply:
Hmm, this will mean that codeblock messages will still have annoying newline then.
The solution would be check if previous symbol is ` ?

Reply:
how is it annoying when it makes no difference?

Reply:
![image](https://user-images.githubusercontent.com/22667809/99155644-c6790000-26ca-11eb-887a-2b7a113c7617.png)
compare this to screenshot in PR message

Reply:
ah, I see. Was hard to find it at first glance